src/GenAPI/Microsoft.DotNet.GenAPI/Filters/ImplicitSymbolsFilter.cs

```diff
@@ -0,0 +1,31 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using Microsoft.CodeAnalysis;
+using Microsoft.DotNet.ApiSymbolExtensions;
+
+namespace Microsoft.DotNet.GenAPI
+{
+    internal class ImplicitSymbolsFilter : ISymbolFilter
+    {
+        /// <inheritdoc />
+        public bool Include(ISymbol member)
```

Comment:
Instead of inheritdoc, can you please add a triple slash xml comment to explain what this filter does? Just from looking at the code, I have no idea why this filter is required for GenAPI.